src/components/modes/teacher/BackgroundToggle.tsx

```diff
@@ -119,7 +124,8 @@ const BackgroundToggle = () => {
     <>
       <ToggleContainer>
         <Typography
-          sx={{ fontSize: '1.1em', color: toggleDisabled ?? grey[500] }}
+          sx={{ fontSize: '1.1em' }}
+          style={{ color: toggleDisabled ? grey[500] : 'black' }}
```

Comment:
You can use `color=xxx` `fontSize=xxx` since it follows the `system` properties https://mui.com/material-ui/react-typography/#system-props